docs/virtual_machines/net_binding_plugins/network_binding_plugins.md

```diff
@@ -0,0 +1,227 @@
+# Network Binding Plugins
+[v1.1.0, Alpha feature]
+
+A modular plugin which integrates with Kubevirt to implement a
+network binding.
+
+## Overview
+
+### Network Connectivity
+In order for a VM to have access to external network(s), several layers
+need to be defined and configured, depending on the connectivity characteristics
+needs.
+
+These layers include:
+- Host connectivity: Network provider.
```

Comment:
You must add an empty line above each list. Check the rendered page to make sure that everything shows up as expected

Reply:
Done